src/modal/modal.scss

```diff
@@ -28,9 +66,12 @@
     z-index: 1000;
     max-width: 50%;
     min-width: 400px;
+    max-height: 100vh;
     border-radius: $iui-border-radius;
     box-shadow: $iui-elevation-24;
     padding: $iui-baseline $iui-m;
+    box-sizing: border-box;
+    will-change: transform;
```

Comment:
It only applies to full page modal as default one has only opacity animation.

Reply:
Also, I see it is causing this strange bug https://github.com/iTwin/iTwinUI/pull/489#issuecomment-1068816612. Because when removed, modal shows everything clearly.

Reply:
And it only causes the issue in regular modal, so definitely move it to full modal CSS.

Reply:
Done